blocklylib-vertical/src/main/java/com/google/blockly/android/ui/vertical/BlockView.java

```diff
@@ -229,6 +242,15 @@ public ColorFilter getColorFilter() {
     }
 
     /**
+     * Called when a block's inputs, fields, comment, or mutator is/are updated, and thus the
+     * shape may have changed.
+     */
+    protected void onBlockStructureUpdated() {
+        // TODO(AnmAtAnm): Mark view for full recalc?
```

Comment:
Depending what changed you may have to rebuilt the entire view (if inputs, fields, or connections were added/removed). This is fine as a placeholder for this CL.